### ISSUE TITLE
Certain video player command line arguments can't be passed with -a, marked "ambiguous"

### ISSUE THREAD

**Issue by Hubcapp** [streamlink/streamlink#1970]
## Bug Report

- [x] This is a bug report and I have read the contribution guidelines.

### Description

A streamer I was watching had poor audio mixing, and decided that it was fine the way it was. I googled and found some command line option for mpv that would apply an audio filter to normalize it in real time: `--af=lavfi="[alimiter=10:1:1:5:8000]",volume=softclip=yes`. Unfortunately, it is unusable within Streamlink due to an apparent conflict.

### Expected / Actual behavior

**Expected:**
Able to pass any command line arguments I want inside the -a flag.

**Actual:**
Streamlink exits without attempting to open the stream. Says that mpv's `--af` flag conflicts with Streamlink's `--afreeca-username` and `--afreeca-password`

### Reproduction steps / Explicit stream URLs to test

Attempt to use mpv's  `--af` flag.

`$ streamlink -p mpv --title '{author} -!- {title} -!- {game}' --loglevel debug -a '--af=lavfi="[alimiter=10:1:1:5:8000]",volume=softclip=yes {filename}' https://www.twitch.tv/summoningsalt 480p`

### Log output

```
usage: streamlink [OPTIONS] <URL> [STREAM]
streamlink: error: ambiguous option: --af=lavfi="[alimiter=10:1:1:5:8000]",volume=softclip=yes {filename} could match --afreeca-username, --afreeca-password
```

**Comment by beardypig** [comment]
Thanks for the detailed report. This is a duplicate of #971.

As a work around you might be able to do 

```console
$ streamlink -p 'mpv --af=lavfi="[alimiter=10:1:1:5:8000]",volume=softclip=yes' --'{author} -!- {title} -!- {game}' https://www.twitch.tv/summoningsalt 480p
```

**Comment by Hubcapp** [comment]
Ah. Sorry for the duplicate.

After I grepped for the text "ambiguous option" and didn't find it in the streamlink codebase, I knew it was going to be a hard bug to solve... But I didn't think it was a bug over a decade old.

I may try that, stuffing the cli into the -p flag. My work-around last night was to remove the code that added the `--plugin-username` & `--plugin-password` flags, since I don't use them.

Closing as duplicate.
